src/http-spec.ts

```diff
@@ -3,6 +3,14 @@ import { INode, INodeExample, INodeExternalExample } from './graph';
 import { ISchema } from './schemas';
 import { IServer } from './servers';
 
+/**
+ * HTTP Service
+ */
+
+export interface IHttpService extends INode {
```

Comment:
Suggest adding the following, pulled from the oas3 spec:

```
description?: string
termsOfService?: string
```

Reply:
Ah, right - naw let's hold on the rest of the info object until we have a better sense. In fact, can probably just leave as is and not add `termsOfService` until we need it. So, good to go 👍 .